tutorials/Native_distributions_and_local_execution/README.md

```diff
@@ -5,11 +5,18 @@
 In this tutorial, we'll show you how to create native distributions (installers/packages) for all the supported systems.
 We will also demonstrate how to run an application locally with the same settings as for distributions.
 
-## Gradle plugin
+## Available tools
+
+There are two tools available for packaging desktop applications:
+
+1. [jpackage](https://openjdk.java.net/jeps/343), which comes with the JDK.
```

Comment:
We consider `jpackage` an implementation detail. Also we already have some code, that does not use `jpackage` at all. I suggest to rephrase to something like: "Compose Multiplatform Gradle Plugin provides basic packaging, obfuscation (via ProGuard), and signing (currently, macOS only) out-of-the-box."